src/app/collection/pages/collection-ncyte/resource/resource.component.scss

```diff
@@ -0,0 +1,47 @@
+@import '~_vars.scss';
+@import '../../../collection-globals.scss';
+
+.section-resource{
+    display: flex;
+    flex-wrap: wrap;
+    font-size: $normal;
+    margin-top: 100px;
+    flex-basis: 100vw;
+    text-align: center;
+}
+.logo-tech-ncyte{
+    width: 300px;
+}
+a{
+    text-decoration: none;
+    cursor: pointer;
+    color: black;
```

Comment:
This should be a dark grey color. Black is typically avoided in web development because it is very dark against white or light colored backgrounds.